### PR TITLE
port NidecBrushless.java

### PR DESCRIPTION
**Pull request by ariovistus**

### CODE REVIEW

File: wpilib/wpilib/nidecbrushless.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        self.isInverted = False
+        self.disabled = False
+
+        hal.report(hal.UsageReporting.kResourceType_NidecBrushless, pwmChannel)
+        self.setName("Nidec Brushless", pwmChannel)
+
+    def free(self):
+        """Free the resources used by this object."""
+        super().free()
+        self.dio.free()
```
Comment (ariovistus) [inline wpilib/wpilib/nidecbrushless.py]: right, finalizer pattern

Comment (virtuald) [reply wpilib/wpilib/nidecbrushless.py]: If you're not directly allocating a hal resource, then I don't think a finalizer is necessary. Let garbage collection do it's thing.

File: wpilib/wpilib/nidecbrushless.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        self.pwm.free()
+
+    def set(self, speed):
+        """ 
+        Set the PWM value.
+    
+        The PWM value is set using a range of -1.0 to 1.0, appropriately scaling the value for the FPGA.
+
+       :param speed: The speed value between -1.0 and 1.0 to set.
```
Comment (auscompgeek) [inline wpilib/wpilib/nidecbrushless.py]: Indentation doesn't match.

File: wpilib/wpilib/nidecbrushless.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        self.pwm.setDisabled()
+
+    def enable(self):
+        """
+        Re-enable the motor after :meth:`disable` has been called.  
+        The :meth:`set` function must be called to set a new motor speed.
+        """
+        self.disabled = False
+        
+
```
Comment (auscompgeek) [inline wpilib/wpilib/nidecbrushless.py]: One too many newlines here I think.

File: wpilib/wpilib/nidecbrushless.py (hunk 2 above)
Comment (virtuald) [reply wpilib/wpilib/nidecbrushless.py]: @ariovistus are you converting docstrings by hand? That sounds painful, checkout `devtools/convert_javadoc.html`

**Pull request merged**